src/moveit_visual_tools_demo.cpp

```diff
@@ -62,26 +68,26 @@ class VisualToolsDemo
   /**
    * \brief Constructor
    */
-  VisualToolsDemo()
+  VisualToolsDemo(const rclcpp::Node::SharedPtr& node)
+  : node_(node)
   {
-    visual_tools_.reset(new moveit_visual_tools::MoveItVisualTools("world", "/moveit_visual_tools"));
+    visual_tools_.reset(new moveit_visual_tools::MoveItVisualTools(node_, "world", "/moveit_visual_tools"));
     visual_tools_->loadPlanningSceneMonitor();
     visual_tools_->loadMarkerPub(true);
     visual_tools_->loadRobotStatePub("display_robot_state");
     visual_tools_->setManualSceneUpdating();
 
     robot_state_ = visual_tools_->getSharedRobotState();
     jmg_ = robot_state_->getJointModelGroup(PLANNING_GROUP_NAME);
-
     // Allow time to publish messages
-    ros::spinOnce();
-    ros::Duration(0.1).sleep();
+    rclcpp::spin_some(node);
+    rclcpp::sleep_for(std::chrono::milliseconds(1000));
```

Comment:
This changes 100ms to 1000ms. Is that intentional? Also, consider using chrono literals `100ms`

Reply:
same for following sleeps

Reply:
That is intentional, 100ms was way too short to see what was happening in the demo. 1 second seemed better